packages/react/src/Grid/Grid.stories.tsx

```diff
@@ -0,0 +1,65 @@
+import {ComponentMeta, ComponentStory} from '@storybook/react'
+import React from 'react'
+import {Grid, GridProps, GridColumnIndex} from './Grid'
+
+export default {
+  title: 'Components/Grid',
+  component: Grid,
+  args: {
+    as: 'div',
+    gap: 'normal',
+    enableOverlay: true,
+    columns: [6, 6]
+  },
+  argTypes: {
+    gap: {control: 'radio', options: [undefined, 'condensed', 'normal', 'spacious'], defaultValue: undefined},
+    gutters: {control: 'radio', options: [undefined, 'condensed', 'normal', 'spacious'], defaultValue: undefined},
```

Comment:
we can remove these two from the playground

Reply:
thanks 🙏 , removed